docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,153 @@
+giving a correct DNS response for an authoritative name server.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Summary
+
+Message Tag outputted           | Level   | Arguments            | Description of when message tag is outputted
+:-------------------------------|:--------|:---------------------|:--------------------------------------------
+SPF1_FOUND_AND_VALID            | INFO    |                      | The domain publishes a valid SPF version 1 policy.
+SPF1_LOOKUP_LIMIT_EXCEEDED      | ERROR   |                      | Evaluating the published SPF version 1 policy would require more than 10 DNS lookups.
+SPF1_MORE_THAN_ONE_SAME_MODIFIER | ERROR  | modifier             | The published SPF version 1 policy contains the same modifier (redirect or exp) more than once.
+SPF1_MULTIPLE_RECORDS           | ERROR   |                      | The domain publishes more than one SPF version 1 policy.
+SPF1_REDIRECT_AND_ALL           | WARNING |                      | The published SPF version 1 policy contains both the "all" keyword and a redirect modifier.
+SPF1_REDIRECT_NOT_AT_END        | WARNING |                      | The published SPF version 1 policy contains a redirect modifier, but it does not appear at the end of the string.
+SPF1_SYNTAX_ERROR               | ERROR   |                      | The published SPF version 1 policy has a syntax error.
+SPF_NOT_CONFIGURED              | INFO    |                      | No SPF policy was found.
+SPF_UNEXPECTED_ERROR            | ERROR   | rcode                | The server responded with an unexpected RCODE when looking up SPF records.
+SPF_WRONG_VERSION               | WARNING | version              | The domain publishes an SPF policy, but its version number is invalid.
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [argument list].
+
+## Test procedure
+
+1. Create a TXT query for the apex of the *Child Zone* and send it over UDP to
+   the name server.
+
+2. If the name server responds with an RCODE different from NOERROR, then
+   output *[SPF_UNEXPECTED_ERROR]* and terminate the test.
+
+3. If no TXT records begin with the string `v=spf`, output
+   *[SPF_NOT_CONFIGURED]* and terminate the test.
+
+4. If at least one TXT records starts with the string `v=spf`, but none
+   specify an SPF version 1 policy, then output *[SPF_WRONG_VERSION]*.
+
+5. Select all TXT records either equal to the string `v=spf1` or starting with
+   `v=spf1` followed by a space. If there are two or more such records, then
+   output *[SPF1_MULTIPLE_RECORDS]* and terminate the test. The next steps
+   assume there is exactly one such TXT record.
+
+6. If the text does not pass the syntax check for SPF version 1 records,
```

Comment:
Please make "pass the syntax check" a link to the terminology section. See how "Null MX" is handled in https://github.com/zonemaster/zonemaster/blob/develop/docs/specifications/tests/Zone-TP/zone09.md?plain=1